backend/server/src/controllers/tournamentController.ts

```diff
@@ -23,6 +27,16 @@ export class TournamentController extends Controller {
     return await this.service.getTournamentById(id);
   }
 
+  @Security("jwt")
+  @Get("{id}/details")
```

Comment:
I think that specifying this endpoint as just `{id}` is sufficient and more REST-like